addons/website_event_booth_sale/models/sale_order.py

```diff
@@ -8,51 +8,69 @@
 class SaleOrder(models.Model):
     _inherit = 'sale.order'
 
-    def _cart_find_product_line(self, product_id=None, line_id=None, **kwargs):
+    def _cart_find_product_line(
+        self, product_id=None, line_id=None,
+        event_booth_pending_ids=None, **kwargs
+    ):
         """Check if there is another sale order line which already contains the requested event_booth_pending_ids
         to overwrite it with the newly requested booths to avoid having multiple so_line related to the same booths"""
-        self.ensure_one()
-        lines = super(SaleOrder, self)._cart_find_product_line(product_id, line_id)
-        if line_id:
+        lines = super()._cart_find_product_line(product_id, line_id, **kwargs)
```

Comment:
I usually pass all values to avoid "loosing information" but I guess both approaches have their advantages. 
Just wanted to see if it was intentional or not, you can keep it like that.

Reply:
Strange to alter given parameters indeed ... normally input should never be modified, methods should work on local copies when possible in order to let all overrides access to original input. Not critical here as it is not likely that someone will use ``_cart_find_product_line`` with booth information somewhere between website_event_booth_sale and website_sale, but I would also propagate them ^^ .